docs/release_notes_v2_2_1.md

```diff
@@ -12,7 +12,9 @@ Check out the newest features and enhancements.
 
 
 ## General Improvements
-
+Enabled Elasticsearch queries to be built on base searches of any kind. Users can now save time by
+creating reusable candidate searches to pre-filter on the basis of commonly applicable criteria 
+such as organisational countries of operation and immigration pathway eligibility.
```

Comment:
Hey @samschlicht this is such a useful feature that I think it deserves a full page with screenshots / video showing how to use it. Well done!

Reply:
Thanks, @sadatmalik - will do!